src/main/resources/templates/login.html

```diff
@@ -9,8 +9,14 @@
 
 <body>
   <div layout:fragment="content">
-    <p>Please log in to use Dim Sum</p>
-    <a th:href="@{/saml2/authenticate/dimsum}">Login</a>
+    <div class="max-w-lg mx-auto my-20 flex flex-col rounded-lg px-8">
```

Comment:
Was this meant to have a border? `rounded-lg` doesn't seem to have any effect on its own